_assets/scripts/run_unit_tests.sh

```diff
@@ -124,3 +131,12 @@ if [[ "${UNIT_TEST_COUNT}" -gt 1 ]]; then
     fi
   done
 fi
+
+# Gather test coverage results
+echo "mode: atomic" > c.out
+grep -h -v "^mode:" ./**/*.coverage.out >> c.out
+rm -rf ./**/*.coverage.out
```

Comment:
Not sure if it's related, there are some built-in tools for combining results:
https://docs.codeclimate.com/docs/configuring-test-coverage#multiple-test-suites
JFYI

Reply:
yeah I've seen that, but had issues using it 🙁 
Goggled for solution and ended up doing it manually as suggested somewhere.